.github/workflows/copilot-setup-steps.yml

```diff
@@ -0,0 +1,86 @@
+name: "Copilot Setup Steps"
+inputs:
+  robot-version:
+    description: 'Version of ROBOT tool to install'
+    required: false
+    default: 'v1.9.7'
+
+
+# Automatically run the setup steps when they are changed to allow for easy validation, and
+# allow manual testing through the repository's "Actions" tab
+on:
+  workflow_dispatch:
+  push:
+    paths:
+      - .github/workflows/copilot-setup-steps.yml
+  pull_request:
+    paths:
+      - .github/workflows/copilot-setup-steps.yml
+
+jobs:
+  # The job MUST be called `copilot-setup-steps` or it will not be picked up by Copilot.
+  copilot-setup-steps:
+    runs-on: ubuntu-latest
+
+    # Set the permissions to the lowest permissions possible needed for your steps.
+    # Copilot will be given its own token for its operations.
+    permissions:
+      # If you want to clone the repository as part of your setup steps, for example to install dependencies, you'll need the `contents: read` permission. If you don't clone the repository in your setup steps, Copilot will do this for you automatically after the steps complete.
+      contents: read
+
+    # You can define any steps you want, and they will run before the agent starts.
+    # If you do not check out your code, Copilot will do this for you.
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+
+    - name: Create tools directory
+      run: mkdir -p ${{ github.workspace }}/tools
+      shell: bash
+
+    - name: Cache ROBOT JAR files
+      uses: actions/cache@v4
+      with:
+        path: ~/.jar-cache
+        key: ${{ runner.os }}-robot-v1.9.7
+        restore-keys: ${{ runner.os }}-robot-
+
+    - name: Download ROBOT JAR if not cached
+      run: |
+        mkdir -p ~/.jar-cache
+        if [ ! -f ~/.jar-cache/robot.jar ]; then
+          curl -L https://github.com/ontodev/robot/releases/download/v1.9.7/robot.jar -o ~/.jar-cache/robot.jar
```

Comment:
ODK version 1.5.4 uses ROBOT version 1.9.8. is something incompatible with this ROBOT version?

Reply:
Or to ask it another way - why not just use ODK instead of downloading the JAR file?

Reply:
If you can figure out how to do that!

https://github.com/ai4curation/aidocs/pull/5